bind-all.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @module 101/bind-all
+ */
+
+var isFunction = require('./is-function');
+var keysIn = require('./keys-in');
+/**
+ * Bind a passed object methods.
+ * If methods name to bing are not specified, all the object methods are binded
+ * @function module:101/bind-all
+ *
+ * @param {object} object - object to bind
+ * @param {array|string} [methods] - array or space-separated string containing the names of the methods to bind
+ * @return {object} the binded object
+ */
+module.exports = bindAll;
+
+function bindAll (object, methods) {
+  var keys;
+  if (methods) {
+    // if we passed an array, set keys to the passed array
+    if (Array.isArray(methods)) {
+      keys = methods;
+    } else if (typeof methods === 'string') {
+      // if a string was paased, split it by spaces
+      keys = methods.split(/,\s|[,\s]/);
```

Comment:
IMO string should not be used to represent a list for the sake of saving keystrokes. My guess is that they've added it to support users with data in this format.

This type of flexibility is common in JS utils, however splitting a string by spaces is a trivial operation. Also I don't think users would assume that `bindAll` accepts a space delimited string, as it doesn't seem very natural to me.

I don't mind incompatibilities with underscore/lodash IMO their methods are very flexible/robust which is great, but I prefer predictable, explicit behavior versus overloading utils.

Thoughts @bkendall @stoeffel